airbyte-integrations/connectors/source-harvest/source_harvest/source.py

```diff
@@ -85,6 +85,10 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         auth = self.get_authenticator(config)
         replication_start_date = pendulum.parse(config["replication_start_date"])
         from_date = replication_start_date.date()
+        replication_end_date = config.get("replication_end_date")
```

Comment:
Yep, this is done only for the testing purposes. Previous builds failed due to a timeout when one of the incremental test cases was run. I could increase the timeout but I don't think it's a good idea. Another option is to decrease a time range for the incremental sync tests - so an upper date/time bound is needed here. But since it's done for testing purpose only - spec is not modified and handling this argument should be done in a safe way

Reply:
Could you please add `replication_end_date` to the spec with `airbyte_hidden: true`? I'd love the SAT config to match the spec as much as possible.

Reply:
done